analysis/paladinholy/src/modules/shadowlands/legendaries/MaraadsAnalysis.tsx

```diff
@@ -0,0 +1,135 @@
+import { formatPercentage } from 'common/format';
+import SPELLS from 'common/SPELLS';
+import HIT_TYPES from 'game/HIT_TYPES';
+import { SpellLink } from 'interface';
+import Analyzer, { SELECTED_PLAYER, Options } from 'parser/core/Analyzer';
+import Events, { CastEvent, HealEvent } from 'parser/core/Events';
+import { ThresholdStyle, When } from 'parser/core/ParseResults';
+
+const OVERHEAL_THRESHOLD = 0.5;
+
+class MaraadsAnalysis extends Analyzer {
+  totalCasts = 0;
+  overheal = 0;
+  castsOverhealed = 0;
+  LODcasts = 0;
+
+  constructor(options: Options) {
+    super(options);
+    this.active = this.selectedCombatant.hasLegendaryByBonusID(SPELLS.MARAADS_DYING_BREATH.bonusID);
+    if (!this.active) {
+      return;
+    }
+    this.addEventListener(
+      Events.cast.spell(SPELLS.LIGHT_OF_THE_MARTYR).by(SELECTED_PLAYER),
+      this.cast,
+    );
+    this.addEventListener(
+      Events.heal.spell(SPELLS.LIGHT_OF_THE_MARTYR).by(SELECTED_PLAYER),
+      this.heal,
+    );
+    this.addEventListener(
+      Events.cast.spell(SPELLS.LIGHT_OF_DAWN_CAST).by(SELECTED_PLAYER),
+      this.LODcast,
+    );
+  }
+
+  cast(event: CastEvent) {
+    this.totalCasts += 1;
+  }
+
+  LODcast(event: CastEvent) {
+    this.LODcasts += 1;
+  }
+
+  private heal(event: HealEvent) {
+    const totalHeal = event.amount + (event.overheal || 0) + (event.absorbed || 0);
+    const effectiveHeal = event.amount + (event.absorbed || 0);
+    if (event.hitType !== HIT_TYPES.CRIT && effectiveHeal / totalHeal < OVERHEAL_THRESHOLD) {
+      this.castsOverhealed += 1;
+    }
+  }
+
+  get unbuffedCastsSuggestion() {
+    return {
+      actual: 1 - this.LODcasts / this.totalCasts,
+      isGreaterThan: {
+        minor: 0.1,
+        average: 0.2,
+        major: 0.3,
+      },
+      style: ThresholdStyle.PERCENTAGE,
+    };
+  }
+
+  get notEnoughCastsSuggestion() {
+    return {
+      actual: 1 - this.totalCasts / this.LODcasts,
```

Comment:
Since this is a ratio you should probably do some absolute value magic since
38 LoTM and 32 LoD = 1 - 38 / 32 = -.1875
when
38 LoD and 32 LoTM = 1 - 32 / 38 = .15

Which while neither are optimal gameplay one will pass and one will fail to produce a warning message